website/docs/whats-new/release-notes/partials/flight-icons.md

```diff
@@ -0,0 +1,84 @@
+**`npm package: @hashicorp/flight-icons`**
+
+## 2.20.0 <Doc::Badge @type="outlined">Latest</Doc::Badge>
+
+- Added "static" version of the animated icons:
+
+    - `loading-static`
+    - `running-static`
+
+- Added new set of icons for Google services:
+
+    - `google-docs`
+    - `google-forms`
+    - `google-slides`
+    - `google-sheets`
+    - `google-drive`
+
+  - [#1657](https://github.com/hashicorp/design-system/pull/1657) Thanks [@jorytindall](https://github.com/jorytindall)! 
+
+## 2.19.0
+
+**Month Day, Year**
+
+- `Vault Secrets` icon added. [#1638](https://github.com/hashicorp/design-system/pull/1638) Thanks [@hashibot-hds](https://github.com/hashibot-hds)!
+
+## 2.18.0
+
+**Month Day, Year**
+
+- `Vercel` and `Jira` icons added. [#1585](https://github.com/hashicorp/design-system/pull/1585) Thanks [@hashibot-hds](https://github.com/hashibot-hds)! 
+
+- `twitter-x` icon, and older `twitter` icon added. Also the `aws-cdk` and `jfrog` icons added.[#1610](https://github.com/hashicorp/design-system/pull/1610) Thanks [@hashibot-hds](https://github.com/hashibot-hds)!
+
+## 2.17.0
+
+**Month Day, Year**
+
+- `clipboard-x` icon added. [#1532](https://github.com/hashicorp/design-system/pull/1532) Thanks [@hashibot-hds](https://github.com/hashibot-hds)! 
+
+## 2.16.0
+
+**Month Day, Year**
+
+- `Elastic observability` and `New relic` icons added. 
+[#1515](https://github.com/hashicorp/design-system/pull/1515) Thanks [@hashibot-hds](https://github.com/hashibot-hds)!
+
+## 2.15.0
+
+**Month Day, Year**
+
+- `Venafi` service icon added. [#1487](https://github.com/hashicorp/design-system/pull/1487) Thanks [@hashibot-hds](https://github.com/hashibot-hds)! 
+
+## 2.14.0
+
+**Month Day, Year**
+
+- `Robot` icon for Service Principals added. [#1428](https://github.com/hashicorp/design-system/pull/1428) Thanks [@jorytindall](https://github.com/jorytindall)!
+
+## 2.13.1
+
+**Month Day, Year**
+
+- `Connection-gateway` fixed color for icon. [#1420](https://github.com/hashicorp/design-system/pull/1420) Thanks [@hashibot-hds](https://github.com/hashibot-hds)! 
+
+## 2.13.0
+
+**Month Day, Year**
+
+- `Ampersand` icon added. [#1412](https://github.com/hashicorp/design-system/pull/1412) Thanks [@hashibot-hds](https://github.com/hashibot-hds)!
+
+## 2.12.0
+
+**Month Day, Year**
+
+- `css` icons remove since unnused. [#727](https://github.com/hashicorp/design-system/pull/727) Thanks [@Dhaulagiri](https://github.com/Dhaulagiri)! 
+    
+    - _Notice: as an exception, we are releasing this change as minor instead of major for the reason that no one is actually using these files._
+
+- `Figma` and `Loom` icons added. [#689](https://github.com/hashicorp/design-system/pull/689) Thanks [@hashibot-hds](https://github.com/hashibot-hds)! 
```

Comment:
```suggestion
- Removed unused css icons. [#727](https://github.com/hashicorp/design-system/pull/727) Thanks [@Dhaulagiri](https://github.com/Dhaulagiri)! 
    
    - **Note:** We are releasing this change as minor instead of major becaues no one is actually using these files.
    
- Added a new set of service icons: 

    - `figma`
    - `figma-color`
    - `loom`
    - `loom-color`

[#689](https://github.com/hashicorp/design-system/pull/689) Thanks [@hashibot-hds](https://github.com/hashibot-hds)! 
```